integration_tests/src/main/python/spark_session.py

```diff
@@ -111,6 +111,10 @@ def is_before_spark_320():
 def is_before_spark_330():
     return spark_version() < "3.3.0"
 
+# The bug SPARK-37451 only affects the following versions
+def is_neg_dec_scale_bug_version():
```

Comment:
Shouldn't this definition be in the one file that needs it, cast_test.py?  I don't see us needing to check for this one very specific bugfix across tests in general.

Reply:
Sure, I can move this but I wasn't sure we like accessing `spark_version()` method outside of `spark_session.py`